server/filter.go

```diff
@@ -159,7 +159,7 @@ func (f *snapshotCountFilter) FilterTarget(store *storeInfo) bool {
 // storageThresholdFilter ensures that we will not use an almost full store as a target.
 type storageThresholdFilter struct{}
 
-const storageRatioThreshold = 0.8
+const storageRatioThreshold = 0.2
```

Comment:
It's for checking if ok to move data to the store. It should depend on available space.